ramlfications/utils/types.py

```diff
@@ -119,17 +119,10 @@ def parse_example(name, node):
     data = dict(name=name, value=node)
     if name:
         data["display_name"] = name
-    if isinstance(node, dict):
-        # Might have a 'value' key; adds a layer.
-        if "value" in node:
-            data["value"] = node["value"]
-            if "description" in node:
-                data["description"] = BaseContent(node["description"])
-            if "displayName" in node:
-                data["display_name"] = node["displayName"]
-            if "strict" in node:
-                data["strict"] = node["strict"]
-    if "display_name" not in data:
-        data["display_name"] = name
+    if isinstance(node, dict) and "value" in node:
```

Comment:
couldn't you also remove the `and "value" in node:` and do similarly `data["value"] = node.get("value")?  I know there _has_ to be a value, but that can be caught in validation.